Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/Constants.cs

```diff
@@ -1,13 +1,20 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
```

Comment:
Remove and sort `using` statements.  None of these should be necessary.

Reply:
Fixing it now.